src/app.js

```diff
@@ -28,6 +28,13 @@ export class OSMap extends HTMLElement {
   }
 
   connectedCallback() {
+    if (!mapboxgl.supported()) {
+      this.querySelectorAll(".os-hidden").forEach((el) => {
+        el.classList.remove("os-hidden");
+      });
```

Comment:
This works only if JavaScript is enabled.  How can we display fallback content if JavaScript is disabled?